plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/rows/functions.tsx

```diff
@@ -59,9 +90,37 @@ function getStatusBadges( subscription: Subscription ): StatusBadge[] {
 		badges.push( {
 			text: __( 'Expired', 'woocommerce' ),
 			level: StatusLevel.Error,
-			explanation: __(
-				'To receive updates and support, please renew your subscription.',
-				'woocommerce'
+			explanation: createInterpolateElement(
```

Comment:
`createInterpolateElement` looks awesome for this complex translatable strings. TIL!